### ISSUE TITLE
Storybook 6.5.5 - Angular - No show Controls & Actions on Google Chrome (Microsoft Edge is working) with MDX

### ISSUE THREAD

**Issue by jserra91** [storybookjs/storybook#18333]
**Describe the bug**
I am using Storybook (6.5.5). I created a story with MDX and it's working correctly on Microsoft Edge. But with Google Chrome when you click on Canvas (top bar) to display the story the bottom bar (Actions and Controls) is not displayed. Only I can reproduce the issue with Google Chrome

I think that is the CSS issue because in Microsoft Edge have one height property inside of the one style and with Chrome no. If you edit manually this property (add), automatically is displayed well.

https://ibb.co/FxTWjPz -> Microsoft Edge
https://ibb.co/LZzwDz3 -> Google Chrome

**To Reproduce**
Create a MDX file (story) with the essentials addon and check that with Microsoft Edge is working and with Google Chrome no.

The use case is to create a workspace and a library with Angular CLI.

Repo: https://github.com/jserra91/sb-angular13-wrf

Config:
story.stories.mdx
```
import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';

import {ButtonIconComponent } from './button-icon.component';

import { CommonModule } from '@angular/common';

<Meta
  title="Atoms/wrf-button-icon"
  parameters={{
    color: {
      values: [
        { name: 'primary', value: 'primary' },
        { name: 'secondary', value: 'secondary' }
      ]
    }
  }}
  component={ButtonIconComponent }/>

export const Template = (args) => ({ props: args });

wrf-button-icon

Button HTML element contains a label and icon element.

<Canvas>
  <Story
    name="Default"
    args={{
      label: "Save",
      color: "primary"
    }}>
    {Template.bind({})}
  </Story>
</Canvas>
```

main.js
```
module.exports = {
    "stories": [
      "../projects/**/*.stories.mdx",
      "../projects/**/*.stories.@(js|jsx|ts|tsx)"
    ],
    "addons": [
      "@storybook/addon-links",
      "@storybook/addon-essentials",
      "@storybook/addon-docs"
    ],
    "framework": "@storybook/angular",
    "core": {
      "builder": "@storybook/builder-webpack5"
    }
  }
```
preview.js
```
import { setCompodocJson } from "@storybook/addon-docs/angular";
import docJson from "../documentation.json";
setCompodocJson(docJson);

export const parameters = {
  actions: { argTypesRegex: "^on[A-Z].*" },
  controls: {
    matchers: {
      color: /(background|color)$/i,
      date: /Date$/,
    },
  },
  docs: { inlineStories: true },
}
```
tsconfig.json
```
{
    "extends": "../tsconfig.app.json",
    "compilerOptions": {
      "types": [
        "node"
      ],
      "allowSyntheticDefaultImports": true
    },
    "exclude": [
      "../src/test.ts",
      "../src/**/*.spec.ts",
      "../projects/**/*.spec.ts"
    ],
    "include": [
      "../src/**/*",
      "../projects/**/*"
    ],
    "files": [
      "./typings.d.ts"
    ]
  }
```
typings.d.ts
```
declare module '*.md' {
    const content: string;
    export default content;
  }
```

**System**
Environment Info:

  System:
    OS: Windows 10 10.0.19044
    CPU: (16) x64 11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz
  Binaries:
    npm: 6.14.13 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (101.0.1210.53)
  npmPackages:
    @storybook/addon-actions: 6.5.5 => 6.5.5
    @storybook/addon-essentials: 6.5.5 => 6.5.5
    @storybook/addon-interactions: 6.5.5 => 6.5.5
    @storybook/addon-links: 6.5.5 => 6.5.5
    @storybook/angular: 6.5.5 => 6.5.5
    @storybook/builder-webpack5: 6.5.5 => 6.5.5
    @storybook/manager-webpack5: 6.5.5 => 6.5.5
    @storybook/testing-library: ^0.0.11 => 0.0.11

**Comment by DzmVasileusky** [comment]
@jserra91 I have experienced similar issue.
In my case something was broken when I have experimented with addons visibility
Try to clear site data for storybook page

**Comment by shilman** [comment]
@jserra91 Do you a have a reproduction repo you can share? If not, can you create one? See [how to create a repro](https://storybook.js.org/docs/react/contribute/how-to-reproduce). Thank you! 🙏

**Comment by jserra91** [comment]
Hello @DzmVasileusky, I thought the same. In fact, when I read Shilman comment, I found exactly the use case that reproduces the error 😄 

@shilman
https://github.com/jserra91/sb-angular13-wrf

Note:
The use case is to create a workspace and a library with Angular CLI.

**Comment by DzmVasileusky** [comment]
@jserra91 It seems storybook 6.5 is very unstable with Angular 13, I've created a repro in another issue as well.
But I'm also using NX
@shilman I am kinda new with Storybook is Angular + Storybook problems are common or was it just popped up recently?

**Comment by rubenswyerstcs** [comment]
I'm actually seeing this with React on a project as well. I've been looking for a fix to not seeing the addons section on Google Chrome and stumbled upon this issue. I checked Firefox and it's working there.

**Comment by jserra91** [comment]
I close the issue. Is fixed on version 6.5.9.

Thank you,
